### DISCUSSION TITLE
Does your IBM I extension support browse history

### DISCUSSION THREAD

**Discussion by JoeCharlesMorgan** [codefori/vscode-ibmi#1528] [General]
Would like to know if the extension supports browse history.  Reason for asking is that I may open the same source for browsing multiple times a day.  Don't want to leave everything I open as it clutters VS though.  And also get weary from always having to use CTL+ALT+O to then type the source path time and again.  We support multiple releases of the our software across a number of different libraries so a filter is not too useful. Thanks!

**Comment by worksofliam** [comment]
@JoeCharlesMorgan Not history as such, but Ctrl+Alt+O to open a file does not require you to enter a full path, but you can just use it as a filter for previously listed sources.

**Comment by JoeCharlesMorgan** [comment]
Barry,
When I use Ctl+Alt+O I see a list of source members but for a different library of which none have I ever opened.  How can I use this command to create a filter for previously opened source members?  Thanks!

On Sep 7, 2023, at 1:43 PM, barry ***@***.***> wrote:

﻿

@JoeCharlesMorgan<https://github.com/JoeCharlesMorgan> Not history as such, but Ctrl+Alt+O to open a file does not require you to enter a full path, but you can just use it as a filter for previously listed sources.
